css/media-query/min-width-1025px.css

```diff
@@ -0,0 +1,120 @@
+.show-lg {
+  display: block;
+}
+
+.d-flex-lg {
+  display: flex;
+}
+
+.big-img img,
+.logo {
+  display: none;
+}
+
+.mobi-nav {
+  height: 49px;
+}
+
+.nav-btm {
+  padding-left: 25px;
+  padding-top: 10px;
+  padding-bottom: 10px;
+}
+
+.main-menu-list {
+  margin-left: 25px;
+}
+
+.top-menu {
+  margin-left: 0;
+  padding-left: 25px;
+}
+
+/* main section */
+
+.intro-art {
+  margin-bottom: 60px;
+}
+
+.intro-art .first-art {
+  height: 70vh;
+  padding-top: 170px;
+  padding-left: 20px;
+}
+
+.intro-art .second-art,
+.intro-art .third-art {
+  /* padding-top: 30px;
+  padding-left: 10px; */
```

Comment:
- [ ] Please remove this commented code.

Reply:
Thank you for taking the time to review our project, Murilo.  We appreciate your detailed eye.

### To the next reviewer:
##### We have:
* Set the yellow bar to shrink at 768px (medium display), instead of 1025px (large display)
* Added a TNW logo in the left image of the top grid at large display
* Aligned the links for the "funds" section to flex at medium display
* Removed commented code from the stylesheet

We look forward to the review.